packages/core/src/components/editable.tsx

```diff
@@ -209,8 +211,24 @@ export function Editable(origProps: EditableProps) {
     onStopComposing: () => setIsComposing(false),
   })
 
-  createEffect(() => {
-    window.document.addEventListener('selectionchange', onDOMSelectionChange)
+  onMount(() => {
+    const window = SolidEditor.getWindow(editor())
+    window.document.addEventListener(
```

Comment:
@Noobulater It's possible this was the culprit of the editor seemingly not unmounting. Also forgot I hadn't merged it, but this fixes an issue where event handlers kept getting added when changing examples and not getting removed.